unitTests/magicbox/SuggestionsManagerTest.ts

```diff
@@ -653,6 +655,8 @@ export function SuggestionsManagerTest() {
                       `Unexpected preview at suggestion #${suggestionId} preview #${previewId}.`
                     );
                     expect(preview.onSelect).toHaveBeenCalled();
+                    expect(blurSpy).toHaveBeenCalled();
```

Comment:
I prefer the the stricter check for the number of times it was called. Combined with the above comment:

```
expect(inputManager.blur).toHaveBeenCalledTimes(1);
```

Also, you should not need to call `reset`.

Reply:
I'll remove the call to `reset` and use this instead.